dbt/include/global_project/macros/materializations/snapshots/snapshot.sql

```diff
@@ -25,16 +25,20 @@
 
   {% set strategy_macro = strategy_dispatch(strategy_name) %}
   {% set strategy = strategy_macro(model, "snapshotted_data", "source_data", config, target_relation_exists) %}
+  {% set dbt_updated_at_data_type = none %}
```

Comment:
I think this line is unnecessary since it will get set below in the `else` clause (or `if` if the condition is met).

Reply:
I removed this line.